dev-go/esc/esc-20170907.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GOLANG_PKG_NAME="esc"
+GOLANG_PKG_IMPORTPATH="github.com/mjibson"
+GOLANG_PKG_VERSION="58d9cde84f237ecdd89bd7f61c2de2853f4c5c6e"
```

Comment:
You can safely use the reselase tarball for v0.1.0 instead of a SHA snapshot --> https://github.com/mjibson/esc/releases

After that, you can mark stable KEYWORDS too.